src/tests/test_deps.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from app.api import crud
+from app.api.deps import get_current_user, get_current_device
+from app.api.schemas import AccessRead, UserRead, DeviceOut
```

Comment:
we do not pass by routes when testing deps; that's why this is not working

Reply:
I could do : 
from app.db import users, devices
but then which attribute are we patching ? we can't say anymore monkeypatch.setattr(devices, "devices", mock_device_table) because devices is not an attribute of the object devices

Reply:
Well, it's kinda tricky to explain, but I designed the new patching system for this specific purpose: it will override the references to each table in both the `app.api.routes.users` and `app.api.routes.devices` routers which is precisely what we want.

And respectively, the `devices` and `users` routers do have a `devices` and `users` attribute, which are their corresponding tables cf. https://github.com/martin1tab/pyronear-api/blob/Issue28/src/app/api/routes/devices.py#L6 and https://github.com/martin1tab/pyronear-api/blob/Issue28/src/app/api/routes/users.py#L6

Agreed we should rename those at some point, but as of now, my suggestion should be working ;)

Reply:
my bad : 
```
from app.api import deps
```